src/rules/no-empty-source/README.md

```diff
@@ -0,0 +1,35 @@
+# no-empty-source
+
+Disallow empty source.
+
+```css
+  ···\n\t
+/**     ↑
+ *  This whitespace */
+```
+
+A source containing only spaces or/and empty lines is considered empty.
+
+The following patterns are considered warnings:
+
+```css
+
+```
+
+```css
+\t\t
```

Comment:
Viewers won't be able to tell the difference between this example and the first, right? Maybe we should format it in a way so they can? E.g. `\t\t`